interface/src/app/gene/search/search.js

```diff
@@ -264,18 +264,21 @@ angular.module('adage.gene.search', [
           for (var gene in $scope.selectedGenes) {
```

Comment:
Aha, I see - so given what @cgreene and @mhuyck were saying about the the performance of ``hasOwnProperty``, what should we do about these other cases? Should we wrap in an ``if`` statement with ``hasOwnProperty``? Should we try to loop some other way?